### DISCUSSION TITLE
Use `message_ix` with Python 3.9 under Anaconda/Miniconda?

### DISCUSSION THREAD

**Discussion by Aprillmylmy** [iiasa/message_ix#621] [Q&A] [answered]
Dear developer,

Due to the compatibility of Python 3.9 with Matlab, is it possible to install messageix version using Python 3.9? I have tried different options, but it feels Python 3.10 is automatically chosen. 

Thanks very much,
Mengyu

**Comment by khaeru** [comment, marked as answer]
Based on https://github.com/iiasa/message_ix/discussions/617#discussioncomment-2864003 —specifically the line:

> python: 3.10.4 | packaged by conda-forge | (main, Mar 24 2022, 17:43:32) [Clang 12.0.1 ]

…it seems you are using message_ix and Python via Anaconda or Miniconda. So, see [“Installing a different version of Python”](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-python.html?#installing-a-different-version-of-python) in the `conda` documentation.
